contracts/governance/Governor.sol

```diff
@@ -55,6 +55,13 @@ abstract contract Governor is Context, ERC165, EIP712, IGovernor {
         _name = name_;
     }
 
+    /**
+     * @dev Function to receive ETH that will be handled by the governor (disabled if executor is a third party contract)
+     */
+    receive() external payable virtual {
```

Comment:
But we are missing the overrides in the timelock modules, right?

Reply:
Not needed, the require takes care of that... hence the @dev message